.travis.yml

```diff
@@ -29,6 +29,7 @@ env:
     - TRAVIS_FLAVOR=mysql FLAVOR_VERSION=latest
     - TRAVIS_FLAVOR=twemproxy FLAVOR_VERSION=latest
     - TRAVIS_FLAVOR=tokumx
+    - TRAVIS_FLAVOR=rabbitmq FLAVOR_VERSION=3.5.0
```

Comment:
I'm trying to keep parity with whatever we're testing on the other repo. Also, while it might be useful to have one that is at latest, to let us know when something has become deprecated or needs to be fixed for a new version, we also need to have pinned versions to ensure there's no regression.